module2/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
+#include <utility>
+#include <vector>
+
+#include <cstdint>
+
+constexpr size_t width = 32;
+constexpr size_t height = 32;
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(std::array<std::array<uint8_t, width>, height>& getGray);
```

Comment:
Have you considered using aliases for longer types std::vector<std::pair<uint8_t, uint8_t>>?